src/components/Pagination.jsx

```diff
@@ -29,13 +30,15 @@ function Pagination({
   };
 
   return (
-    <ul className="wrapper">
-      <li
-        className={classnames("paginationItem", {
-          disabled: false, // change this logic to add disabled class name
-        })}
-      >
-        <button type="button" className="arrowButton left" onClick={onPrevious}>
+    <ul className="wrapper" aria-label="Blog post pagination list">
+      <li className="paginationItem">
+        <button
+          type="button"
+          className="arrowButton left"
+          aria-label="Goto previous page"
+          onClick={onPrevious}
+          disabled={false} // change this line to disable a button.
```

Comment:
Using `disabled` attribute is more conventional, and it is easier to test.
Candidates are expected to do this in their solutions:
```
disabled={currentPage === 1}
```